coral-component-table/src/scripts/Table.js

```diff
@@ -1174,6 +1180,126 @@ const Table = Decorator(class extends BaseComponent(HTMLTableElement) {
     });
   }
 
+  /** @private */  
+  _onKeyboardDrag(event) {
+    const table = this;
+    const row = event.target.closest('tr[is="coral-table-row"]');
+    
+    if (row && table.orderable) {
```

Comment:
1. preventDefault and stopPropagation so that the Space key does not start scrolling the page.
2. Wrap the drag handle button in a span with role="application", to force Windows screen readers into forms mode while dragging.
3. Set the selected state of the drag handle and add `aria-pressed="true"` to better communicate the dragging state.

```suggestion
    if (row && table.orderable) {
      event.preventDefault();
      event.stopImmediatePropagation();

      // Wrap the drag handle button in a span with role="application",
      // to force Windows screen readers into forms mode while dragging.
      const handle = row.querySelector('[coral-table-roworder]');

      if (handle) {
        if (handle.getAttribute('is') === 'coral-button') {
          handle.selected = true;
          handle.setAttribute('aria-pressed', 'true');
        }

        if (event.target === handle && !handle.closest('span[role="application"]')) {
          const span = document.createElement('span');
          span.setAttribute('role', 'application');
          handle.parentNode.insertBefore(span, handle);
          span.appendChild(handle);
          window.requestAnimationFrame(function() {
            handle.focus();
          });
        }
      }

```